cloud/src/meta-service/meta_service.h

```diff
@@ -709,6 +710,28 @@ class MetaServiceProxy final : public MetaService {
     using MetaServiceMethod = void (cloud::MetaService::*)(::google::protobuf::RpcController*,
                                                            const Request*, Response*,
                                                            ::google::protobuf::Closure*);
+    int64_t get_fdb_client_thread_busyness_percent() {
+        //auto now = steady_clock::now();
+        auto now = std::chrono::steady_clock::now();
+        auto duration_s =
+                duration_cast<std::chrono::seconds>(now - buyness_last_update_time_).count();
+        if (duration_s > config::bvar_qps_update_second) {
+            cache_buyness_percent_ = g_bvar_fdb_client_thread_busyness_percent.get_value();
+            buyness_last_update_time_ = now;
+        }
+        return cache_buyness_percent_;
+    }
+
+    int get_dynamic_retry_count() {
+        int64_t busyness_percent = get_fdb_client_thread_busyness_percent();
+        if (busyness_percent > config::retry_disable_busyness_threshold) {
+            return 0;
```

Comment:
add bvar to record reduce and disable count 
consider latency recorder

Reply:
done